langchain4j-core/src/main/java/dev/langchain4j/store/embedding/EmbeddingStore.java

```diff
@@ -55,6 +55,26 @@ public interface EmbeddingStore<Embedded> {
      */
     List<String> addAll(List<Embedding> embeddings, List<Embedded> embedded);
 
+    /**
+     * Removes an embedding from the store.
+     *
+     * @param id The unique identifier of the embedding to be removed.
+     * @return True if the embedding was successfully removed, false otherwise.
+     */
+    default boolean remove(String id) {
```

Comment:
```suggestion
    default void remove(String id) {
```
What about throwing an exception instead (in case of failures)?
Reasoning: users might not check the returned value explicitly and the problem will be unnoticed.
Exception can also provide details about the failure (e.g. network issue, not existing id, etc).

WDYT?